Platform/Intel/MinPlatformPkg/Include/Dsc/MinPlatformFeaturesPcd.dsc.inc

```diff
@@ -7,7 +7,7 @@
 # A board can enable a feature by configuring the PCD in the board DSC file
 # after this file has been included.
 #
-# Copyright (C) 2022 Intel Corporation
+# Copyright (C) 2022 - 2024 Intel Corporation

```

Comment:
Correct me if I am wrong but do we need to update the current year of the file change or just leave the year of file creation?

Reply:
For Tianocore it is optional.